cmd/chartsvc/handler.go

```diff
@@ -217,11 +217,17 @@ func newChartResponse(c *models.Chart) *apiResponse {
 }
 
 func newChartListResponse(charts []*models.Chart) apiListResponse {
+	chartDict := map[string]*apiResponse{}
 	cl := apiListResponse{}
 	for _, c := range charts {
-		cl = append(cl, newChartResponse(c))
+		chartResponse := newChartResponse(c)
+		// The same chart may be present in different repos, filter just one
+		id := (chartResponse.Relationships["latestChartVersion"].Data).(models.ChartVersion).Digest
+		if found := chartDict[id]; found == nil {
```

Comment:
whilst this does work, I think a more correct way of doing this would be:

```
if _, ok := chartDict[id]; !ok {
...
}
```

The use of `ok` instead of `found` is a Go idiom that I see regularly, so I think it's better to use that name.